packages/legend-studio-extension-query-builder/src/components/QueryBuilder_LegendStudioPlugin.tsx

```diff
@@ -226,6 +227,17 @@ export class QueryBuilder_LegendStudioPlugin
                   element,
                 );
                 queryBuilderExtension.queryBuilderState.resetData();
+                if (
```

Comment:
I wonder if we just do it here, is it enough, there are other places as well, such as `getExtraClassViewContextMenuItemRendererConfigurations` or when we create a new query from a class in a data space, etc.

Reply:
I have a concern here. If we do that we should consider resetting the `queryBuilderState` before doing `setClass` otherwise it resets the `queryParameterState`

Reply:
Should I go ahead and make that change everywhere we do that?